testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -556,6 +556,20 @@ public void testMaterializedView()
                         "FROM\n" +
                         "  nation");
 
+        // information_schema.tables (no filtering on table_name so that ConnectorMetadata.listViews is exercised)
+        assertThat(query("SELECT table_name, table_type FROM information_schema.tables WHERE table_schema = '" + schemaName + "'"))
+                .containsAll("VALUES (VARCHAR '" + viewName + "', VARCHAR 'BASE TABLE')");
+
+        // information_schema.views
+        assertThat(computeActual("SELECT table_name FROM information_schema.views WHERE table_schema = '" + schemaName + "'").getOnlyColumnAsSet())
```

Comment:
are there any other columns that we should test?

Reply:
as above